tests/testthat/_snaps/use_extendr.md

```diff
@@ -4,14 +4,125 @@
       use_extendr()
     Message <message>
       v Creating src/rust/src.
-      v Writing file src/entrypoint.c.
-      v Writing file src/Makevars.
-      v Writing file src/Makevars.win.
-      v Writing file src/.gitignore.
-      v Writing file src/rust/Cargo.toml.
-      v Writing file src/rust/src/lib.rs.
-      v Writing wrappers to R/extendr-wrappers.R.
+      v Writing 'src/entrypoint.c'
+      v Writing 'src/Makevars'
+      v Writing 'src/Makevars.win'
+      v Writing 'src/.gitignore'
+      v Writing 'src/rust/Cargo.toml'.
+      v Writing 'src/rust/src/lib.rs'
+      v Writing 'R/extendr-wrappers.R'
       v Finished configuring extendr for package testpkg.
       * Please update the system requirement in DESCRIPTION file.
       * Please run `rextendr::document()` for changes to take effect.
 
+---
```

Comment:
I also added snapshot tests (below) of the templates to make it easier to know that I wasn't breaking anything in this transition